### PR TITLE
Added in python interactive, based off R interactive, modified yml file

### PR DESCRIPTION
**Pull request by Jeffotter**

### CODE REVIEW

File: docs/_user_guide/python.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Running Python Interactively
+
+On the Yens, Python is available via the `python3` command, a version different from the system Python can be chosen via the `module` command.
```
Comment (natalya-patrikeeva) [inline docs/_user_guide/python.md:3]: I don't think we need to mention modules.

File: docs/_user_guide/python.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+You should see:
+    
+```{.yaml .no-copy title="Terminal Output"} 
+Python 3.10.11 (main, May  6 2023, 14:44:19) [GCC 11.3.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>>
+
+```
+
+### Managing Python Software Libraries and Versions
```
Comment (natalya-patrikeeva) [inline docs/_user_guide/python.md:64]: that's not right

**Review by natalya-patrikeeva — CHANGES_REQUESTED** [review]
This needs a major update. We don't use modules for python. Users can use python3 and point to virtual env page. This page can have an example python script and how to run it interactively

File: docs/_user_guide/python.md (hunk 2 above)
Comment (Jeffotter) [reply docs/_user_guide/python.md:64]: What is not right?

Comment (natalya-patrikeeva) [reply docs/_user_guide/python.md:64]: Hey! I mean the whole page. We don't recommend module for python at all

Comment (natalya-patrikeeva) [reply docs/_user_guide/python.md:64]: and you copied the R section but it does not apply to python libraries

**Review by natalya-patrikeeva — APPROVED** [review]
looks good. I'm a little scared we renamed all these files

**Pull request merged**